### PR TITLE
Sydney Update

### PR DESCRIPTION
**Pull request by mattkelly4**
- Heli ops
- ATIS updates
- Files consolidated into single YSSY file

### CODE REVIEW

**Review by alexd-332 — DISMISSED** [review]

File: docs/aerodromes/sydney.md [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+    b) Jets departing 16L via WOL, except during SODPROPS  
+    c) after a go-around, the next departure from that runway
+
+The SY TCU controller can suspend/resume Auto Release at any time, with the concurrence of **SY ADC**.
+
+"Next" Coordination to SY TCU is not required for aircraft assigned a **Procedural SID** and the Standard Assignable Level, or an aircraft assigned a **Radar SID** and assigned a standard assignable heading (see Coordination below).
+
+### SY ADC / SY SMC
+#### Ground Operations
+As per this (link) diagram, **SY ADC** has Responsibility of the Runways, meaning **SY SMC** must always coordinate with **SY ADC** to allow aircraft to cross runways whilst taxiing. **SY SMC** may request, or **SY ADC** may elect, to release certain runways to the **SY SMC** controller, so they may let aircraft cross the runway without coordination (For example, Releasing runway 07/25 to **SY SMC** whilst PROPS are in progress.) This Release may also be cancelled at the controller's discretion.
```
Comment (chrisg06) [inline docs/aerodromes/sydney.md:236]: missing link, or is it supposed to still be a placeholder?

File: docs/terminal/Sydney TCU/operations.md [hunk 2]
```diff
@@ -87,20 +87,19 @@ Aim to vector aircraft to the following Intercept points during instrument PROPS
 When conducting IVAs, aircraft shall not be transferred to **SY ADC** until established on final.
 
 !!! example
-    **SFW:** “BNZ444, Turn left heading 360, join final Runway 34L, from that heading Cleared Independent Visual Approach, report on final” `AIP GEN 3.4`   
+    **SFW:** “BNZ444, Turn left heading 360, join final Runway 34L, from that heading Cleared Independent Visual Approach” `AIP GEN 3.4`   
     **BNZ444:** "Left heading 360, join final Runway 34L, from that heading Cleared Independent Visual Approach, wilco, BNZ444"
```
Comment (chrisg06) [inline docs/terminal/Sydney TCU/operations.md:91]: can remove wilco as report final was removed

File: docs/aerodromes/sydney.md (hunk 1 above)
Comment (mattkelly4) [reply docs/aerodromes/sydney.md:236]: I didn't write this section, I assume it's a placeholder

File: docs/aerodromes/sydney.md [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+When parallel runways are used for departures but inbound traffic levels don't necessitate independent approaches (low traffic levels), the ATIS OPR INFO shall include:  
+`INDEPENDENT PARALLEL DEPARTURES IN PROGRESS`  
+
+When parallel runways are used for departures and the arrival sequence requires independent approaches (high traffic levels), the ATIS OPR INFO shall include:  
+`INDEPENDENT PARALLEL APPROACHES AND DEPARTURES IN PROGRESS`   
+
+When SODPROPS are in operation, the ATIS OPR INFO shall include:  
+`SIMULTANEOUS OPPOSITE DIRECTION PARALLEL RUNWAY OPERATIONS IN PROGRESS`  
+
+When Independent Visual Approaches are run during PROPS, the OPR INFO must also include:
```
[the remark this answers is not in the document]
Comment (mattkelly4) [reply docs/aerodromes/sydney.md:201]: Good pickup, fixed

File: docs/terminal/Sydney TCU/operations.md (hunk 2 above)
Comment (mattkelly4) [reply docs/terminal/Sydney TCU/operations.md:91]: Fixed

**Review by alexd-332 — DISMISSED** [review]

**Review by chrisg06 — APPROVED** [review]

**Review by tylerthetiletiler — APPROVED** [review]

**Comment by mattkelly4** [comment]
Sorry last push for this one, tidied up everything now

**Comment by zachbf** [comment]
@mattkelly4 only thing is that the ATIS should say "DO NOT PASS THRU ASSIGNED RWY CL" not "CENTRELINE" i think.
I've updated the code but the example image would need updating.
i've also fixed the merge conflicts by running `git merge origin/master`

**Comment by mattkelly4** [comment]
> @mattkelly4 only thing is that the ATIS should say "DO NOT PASS THRU ASSIGNED RWY CL" not "CENTRELINE" i think. I've updated the code but the example image would need updating. i've also fixed the merge conflicts by running `git merge origin/master`

Yeah perfect, I'll update the image quickly now.  Cheers

**Review by zachbf — APPROVED** [review]

**Pull request merged**